src/ert/ensemble_evaluator/evaluator.py

```diff
@@ -341,8 +341,9 @@ async def evaluator_server(self) -> None:
             ping_timeout=60,
             ping_interval=60,
             close_timeout=60,
-        ):
+        ) as server:
             await self._done
+            server.close(close_connections=False)
```

Comment:
Was wondering. Is there a way to test this?

Reply:
Added a test